.github/workflows/gradle.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
```

Comment:
Does this use AdoptOpenJDK? The doc for `actions/setup-java@v1` suggests it uses Zulu Community version.

Reply:
It uses Zulu. I'm not too concerned about using that for the toolkit as its based on OpenJDK; but if you or @mbcoder are, I can try to find a premade action using AdoptOpenJDK.